main.go

```diff
@@ -1634,6 +1775,11 @@ func main() {
 			Usage:  "Check repo against released charts",
 			Action: validateRepo,
 		},
+		{
```

Comment:
@adamkpickering 
removed `override-icons` as an option for command here.
It will be called through `auto` passing the flag `--icon`